.github/workflows/review_pull_request.yml

```
@@ -29,16 +29,16 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
-      - name: Run Detekt on CoroutineTemplate
-        working-directory: ./CoroutineTemplate
+      - name: Run Detekt on template
```

Should it be Upper Camel Case 🤔 
```suggestion
      - name: Run Detekt on Template
```

---

@Wadeewee  I'm thinking to keep it lowercase, then it's aligned with the folder name  -> `template`

What do you think? I'm fine with both approaches 🙏

---

@Tuubz I'm not sure, too, but I'm similarly using lowercase. It will be something like this pattern `android-template. `🙏 Actually, I'm okay with both too.